### ISSUE TITLE
Auto extractor files fail at MMAPS

### ISSUE THREAD

**Issue by WebberLai** [TrinityCore/TrinityCore#29609]
### Description

Running the extractor.sh , but last step extract mmaps will stop immediately

### Expected behaviour

Should extractor mmaps with long task

### Steps to reproduce the problem

1. Build project success , generate bin folder
2. Copy bin folder and extractor.sh to game folder
3. Open terminal , enter `cd /Applications/World\ of\ Warcraft` 
4. enter `sh extractor.sh ` , provide client file with ` .`  , server files with the same `.`
5. enter `4) Extract all files`
6. it will stop at last steps extractor mmaps

```
Extracting Movement Maps (this may take a while)

-- Using ./bin/mmaps_generator
-- Copying extracted data to ./data/mmaps
-- Starting in 5 seconds
TrinityCore rev. c4a03b993156 2024-01-23 10:33:00 +0100 (master branch) (MacOSX, Release, Static) (MMAP generator)
<Ctrl-C> to stop.

No such file or directory 
```

### Branch

master

### TC rev. hash/commit

https://github.com/TrinityCore/TrinityCore/commit/c4a03b99315617f7f08b151bffaf3bf14dc238c2

### Operating system

macOS Sonoma 14.2.1

### Custom changes

no custom change

**Comment by WebberLai** [comment]
BTW the WoW game folder just fresh download
not copy from other folder

**Comment by Fabi** [comment]
Can't confirm that.
I built it on latest macOS. copied the binaries to `/Applications/World\ of\ Warcraft/bin` and the extractor.sh (made it executable) to `/Applications/World\ of\ Warcraft`

Did a `cd Applications/World\ of\ Warcraft` and ./extactor.sh

it ends up extracting everything just fine. Using sh extractor.sh too

If you are using the mac case sensitive file system be sure that a lowercase data folder exists since the Wow one is Data. Otherwise I'm not sure what's wrong for you.

tldr: it works

**Comment by WebberLai** [comment]
Perhaps your inference is correct because I do have a "Data" folder inside the "WoW" folder. How can I determine if my macOS is a case-sensitive file system?

**Comment by Fabi** [comment]
> Perhaps your inference is correct because I do have a "Data" folder inside the "WoW" folder

That is fine if you use the default case insensitive mac file system. Otherwise not

**Comment by WebberLai** [comment]
@Fabi  very strange , I check my OS is Fusion driver + APFS(insensitive)
Still fail at mmaps 
others extractor was fine
this is original WoW folder structure
<img width="402" alt="1" src="https://github.com/TrinityCore/TrinityCore/assets/1185079/bc8d2ce7-c023-4cd7-9ecb-200f78aefaeb">

after I run the script 

<img width="643" alt="2" src="https://github.com/TrinityCore/TrinityCore/assets/1185079/d1c75c0f-9448-44e4-a1b2-0e4039ef6ecb">

mmap still empty , does anything I need to check ?

**Comment by syphex97** [comment]
This issue is not limited to linux, im also presented with the exact same problem. It's something with the source code during build

**Comment by CraftedRO** [comment]
you can just download them already extracted here: https://tc.arctium.io/
